portal-api/src/v1/gef/models/facilities.js

```diff
@@ -64,6 +68,10 @@ class Facility {
         this.hasBeenIssued = Boolean(req.hasBeenIssued);
       }
 
+      if (req.hasBeenIssuedAndAcknowledged != null) {
+        this.hasBeenIssuedAndAcknowledged = Boolean(req.hasBeenIssuedAndAcknowledged);
```

Comment:
isn't this supposed to be Boolean here already?

Reply:
added this here for when updating a facility